client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -260,7 +264,7 @@ export default class UpdateChecks extends PureComponent {
 
     // (1) Send request to check for updates
     const responseJSON = await this.updateChecksAPI.checkLatestVersion(
-      config, _getGlobal, updateCheckInfo && updateCheckInfo.latestVersion
+      config, _getGlobal, updateCheckInfo && updateCheckInfo.latestVersion, updateCheckInfo && updateCheckInfo.stagedRollout
```

Comment:
I'm wondering why this is part of the `updateCheckInfo`, not an additional parameter (similar to `silentCheck`). That would also allow us to work without monkey patching :arrow_up:.

Reply:
True, I thought about this. I decided to include it in `updateCheckInfo` because `updateCheckInfo` contains all the parameters send to the UpdateServer#GET API. Since `stagedRollout` is just another parameter, it conceptually belongs there as well IMO.

Reply:
Makes sense.